api/app/main.py

```diff
@@ -388,10 +388,12 @@ async def stats():
 # make a new web10 account
 @app.post("/register_app",include_in_schema=False)
 async def register_app(info:dict):
+    print("HERE'S THE INFO", info)
     if "url" not in info:
         return
     if "http://" in info["url"] or "localhost" in info["url"] or "file://" in info["url"] or "vscode-webview://" in info["url"]:
-        return
+        if not settings.LOCAL_REGISTRATION:
```

Comment:
make sure to commit the kubernetes files to the branch also so they pop up in here, so that there isn't an issue with the deployment